lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -6,5 +6,6 @@
 from . import views
 
 urlpatterns = [
+    url(r'^programs/(?P<programId>[0-9]+)/$', views.view_program_details, name='program_details_view'),
```

Comment:
@schenedx we do need something here in the meantime. It's easy to leave the UUID regex in place for now and change it later if necessary, so I'd do that.